test/core/utils/find-by.js

```diff
@@ -40,4 +40,36 @@ describe('axe.utils.findBy', function () {
   it('should not throw if passed falsey first parameter', function () {
     assert.isUndefined(axe.utils.findBy(null, 'id', 'macaque'));
   });
+
+  it('ignores any non-object elements in the array', function () {
```

Comment:
This test passes even without your changes, might need a different test / setup.